tests/Test/Xrefcheck/TooManyRequestsSpec.hs

```diff
@@ -150,18 +150,45 @@ test_tooManyRequests = testGroup "429 response tests"
                 | otherwise -> toResponse ("" :: Text, serviceUnavailable503)
       infoReverseAccumulatorRef <- newIORef []
       E.bracket (forkIO $ mock429WithGlobalIORef infoReverseAccumulatorRef) killThread $ \_ -> do
-        _ <- verifyLink "http://127.0.0.1:5000/429grandfinale"
+        _ <- verifyLinkDefault "http://127.0.0.1:5000/429grandfinale"
         infoReverseAccumulator <- readIORef infoReverseAccumulatorRef
         reverse infoReverseAccumulator @?=
           [ ("HEAD", tooManyRequests429)
           , ("HEAD", serviceUnavailable503)
           , ("GET", ok200)
           ]
+    , testCase "Returns an error in case of timeout with 0 maximum retries allowed" $ do
+      let prog = Progress{ pTotal = 1
+                          , pCurrent = 1
+                          , pErrorsUnfixable = 1
+                          , pErrorsFixable = 0
+                          , pTaskTimestamp = Nothing
+                          }
+      checkLinkAndProgressWithServer
+        (setOneSecondAllowedTimeout . (cNetworkingL . ncMaxTimeoutRetriesL .~ 0))
```

Comment:
I think the latter option looks much better if we move each new `& <...>` part to next line:
```
\c -> c
  & setOneSecondAllowedTimeout
  & cNetworkingL . ncMaxTimeoutRetriesL .~ 0
```

Reply:
> This sounds good, though I wonder: why in your opinion it would look much better? With do-block this could look similar (without touching the surrounding code):
> 
> ```haskell
> do
>   setOneSecondAllowedTimeout
>   cNetworkingL . ncMaxTimeoutRetriesL .= 0
> ```

Well, true, I like this `do` block more, too.

Reply:
Oh, I see. Feel free to apply whatever then.